tests/build.gradle

```diff
@@ -24,6 +24,17 @@ task testLean(type: Test) {
     exclude '**/*ThrottleTests*'
     exclude '**/MaxActionDurationTests*'
     exclude '**/*ApiGwTests*'
+    exclude '**/*Cli*'
```

Comment:
Hmm, don't think we should exclude the CLI tests until the CLI is ready to be removed from this repo.

Reply:
I believe testLean is the task Travis runs when validating PRs.

Reply:
Because we need to keep the Cli tests inside the repo: we have a stage with both of the cli and rest tests co-existing: WskCliAction and WskRestAction, for example.

If we do not exclude either Cli or Rest, we will end up running the tests twice for the same purpose, which will take longer time than the Travis limit. That is the reason we keep an exclusion here. When Cli tests are decided to remove completely, we can remove the exclusion.